### ISSUE TITLE
Header overlapping profile picture on scrolling down.

### ISSUE THREAD

**Issue by massablaise** [yashmangal112/yashmangal#11]
![image](https://github.com/yashmangal112/yashmangal/assets/104530207/a234cd6d-c011-49e0-9df8-9a8616d1dde8)

> This issue reports a problem where the header section of a website is overlapping with the profile picture when the user scrolls down the page. It's a visual bug and could cause confusion for users.

**Comment by sricharanandra** [comment]
the navbar is sticky so it will stay fixed at the top of the screen even when scrolling. Do you want it to dissapear when you scroll down
?

**Comment by massablaise** [comment]
@sricharanandra it could possibly rather appear in bold color than transparent at the early instance of scrolling. but the overlapping disappears after the page has scrolled past half the height of the profile profile :)

**Comment by sricharanandra** [comment]
@massablaise
![Screenshot 2023-09-30 161323](https://github.com/yashmangal112/yashmangal/assets/114226379/66fd59c6-3681-42bf-b00b-612087e26b68)

![Screenshot 2023-09-30 161256](https://github.com/yashmangal112/yashmangal/assets/114226379/6586b3f1-34b3-4c7a-b275-744c087371d3)

  I think there's an issue with displaying across different viewports, because this is what I see, a white strip when I'm at the top of the page and theme-based color on the navbar once I scroll down a little.

**Comment by sricharanandra** [comment]
It doesn't seem to be overlapping in mine.

**Comment by massablaise** [comment]
@sricharanandra lemme go through the site again and see if it doesn't overlap this time around

**Comment by massablaise** [comment]
same issue still present for me

https://github.com/yashmangal112/yashmangal/assets/104530207/7b6f2e90-6812-4ce3-9655-a4bf0dffea6b

**Comment by sricharanandra** [comment]
Yeah I noticed this too when i started scrolling slowly, I guess it's caused because of the width parameter set for the color change on scroll. If we decrease the scroll distance to activate the navbar color change, it would still overlap but with a different color.
